PC/launcher.c

```diff
@@ -1516,12 +1598,25 @@ process(int argc, wchar_t ** argv)
     else {
         p = argv[1];
         plen = wcslen(p);
+        if (argc == 2) {
+            slen = wcslen(L"-0");
+            if(!wcsncmp(p, L"-0", slen)) /* Starts with -0 */
+                show_python_list(argv); /* Check for -0 FIRST */
```

Comment:
Can we return the exit code from `show_python_list` rather than using `exit()` in it? It's *probably* safe enough (because I'm sure enough people have done it in the past that the MS CRT protects itself against people doing it), but better to return from the entry point than to force an exit.